src/net.cpp

```diff
@@ -2334,7 +2332,7 @@ CNode::CNode(NodeId idIn, ServiceFlags nLocalServicesIn, int nMyStartingHeightIn
     nNextAddrSend = 0;
     nNextInvSend = 0;
     fRelayTxes = false;
-    pfilter = new CBloomFilter();
+    pfilter = MakeUnique<CBloomFilter>();
```

Comment:
can use `std::make_unique` directly instead of `MakeUnique` since we already require c++14